semgrep/semgrep/rule_schema.yaml

```diff
@@ -71,6 +72,7 @@ definitions:
     type: object
     properties:
       pattern:
+        title: Return finding where Semgrep pattern matches exactly
```

Comment:
Another nit: we could ape the descriptions from the [docs schema table](https://semgrep.dev/docs/writing-rules/rule-syntax/#schema) if that's easier/more consistent